packages/side-nav/src/SideNavGroup.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { ulStyleOverrides, LEFT_RIGHT_OFFSET } from './styles';
+
+const sideNavLabelStyle = css`
+  display: block;
+  color: ${uiColors.green.base};
+  font-size: 12px;
+  font-weight: bold;
+  text-transform: uppercase;
+  padding: 5px ${LEFT_RIGHT_OFFSET}px 5px ${LEFT_RIGHT_OFFSET}px;
+`;
+
+interface SideNavGroupProps {
+  /**
+   * Content that will appear inside of SideNavGroup component.
+   */
+  children: React.ReactNode;
```

Comment:
Is there a reason this is a required prop?

Reply:
Nope, I think it can be optional, if you want to use a self closing tab. However, what would that mean in terms of the UI? A group with no members behind a label?

Reply:
In theory – I could go either way here. I suppose there isn't a lot of importance to typing it either way, just that it's perfectly valid to not pass children.